meteor/.meteor/packages

```diff
@@ -1,32 +1,33 @@
 # Meteor packages used by this project, one per line.
+# Check this file (and the other files in this directory) into your repository.
```

Comment:
What's the difference between this file and meteor/.meteor/versions? For example, we have 2 versions specified for blaze-html-templates.

Reply:
great question, I *think* the packages file is the declarative set of intended packages where the versions is the resultant set of everything pulled in including dependencies. Dunno about the version mismatch; that's interesting but doesn't seem to affect functionality.